README.md

```diff
@@ -37,6 +37,7 @@ Version | Kafka Version
 4.1.9   | 2.5.0
 4.1.10  | 2.6.0
 4.1.11  | 2.6.0 (Scala 2.13)
+4.1.12  | 2.8.0
```

Comment:
Let's bump up the version to 4.2.0 here and in `pom.xml` because this is a significant change.